CommonVoice-Data/data/gutenberg/33463.txt

```diff
@@ -1,2 +0,0 @@
-Jules BARDOUX, Directeur.
```

Comment:
it's not empty

Reply:
It's one line. It's not a book. It's not significant nor representative. It's clearly buggy. It's part of the tail of `ls -l`. Should I really argue?

Reply:
ok, this one has really no value